rero_ils/modules/holdings/api.py

```diff
@@ -81,6 +84,27 @@ class Holding(IlsRecord):
             'itty': 'circulation_category'
         }
     }
+    # interval definitions for pattern frequencies
+    # TODO: code the three times a week rdafr:1002 and three times a month
+    # the RDA Frequencies are available here:
+    # http://www.rdaregistry.info/termList/frequency/
+    frequencies = {
+        'rdafr:1001': relativedelta(days=1),  # Daily
+        'rdafr:1002': relativedelta(days=1),  # Three times a week
+        'rdafr:1003': relativedelta(weeks=2),  # Biweekly
+        'rdafr:1004': relativedelta(weeks=1),  # Weekly
+        'rdafr:1005': relativedelta(days=4),  # Semiweekly
```

Comment:
relativedelta supports float you can use `relativedelta(weeks=0.5)` or `relativedelta(days=3.5)` please adapt your configuration accordingly